src/tests/unit/tests/injected/components/command-bar.test.tsx

```diff
@@ -121,13 +135,11 @@ describe('CommandBar', () => {
                 onClickInspectButton: onClickMock.object,
             };
 
-            const wrapper = shallow(<CommandBar {...props} />);
-
-            const button = wrapper.find('.insights-dialog-button-inspect');
+            render(<CommandBar {...props} />);
 
-            button.simulate('click', eventStub);
+            getMockComponentClassPropsForCall(DefaultButton).onClick();
```

Comment:
We should pass in `eventStub` so we can verify that the mock is called properly.
```suggestion
            getMockComponentClassPropsForCall(DefaultButton).onClick(eventStub);
```